packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -348,7 +348,17 @@ class AnimationController extends Animation<double>
   /// after this method is called.
   @override
   void dispose() {
+    assert(() {
+      if (_ticker == null) {
+        throw new FlutterError(
+          'AnimationController.dispose() called more than once.\n'
+          'A given AnimationController for $debugLabel cannot be disposed more than once.'
+        );
+      }
+      return true;
+    });
     _ticker.dispose();
+    _ticker = null;
```

Comment:
Can you make the toString resilient against a null _ticker? (e.g. notice that and add DISPOSED to the details, and don't show info about the ticker)